src/components/Capture/capture.js

```diff
@@ -0,0 +1,239 @@
+import { h, Component } from 'preact'
```

Comment:
It's not IDE. I wrapped the Capture component and put it's public interface i.e. what App consumes in `index.js`. I move the internals to `capture.js` to make the index more readable.